packages/cli/lib/discoverManifests.js

```diff
@@ -14,6 +15,15 @@ export default async function * () {
     },
   })
 
+  const dir = process.cwd()
+  if (hasManifest(dir)) {
+    yield {
+      name: path.basename(dir),
+      manifest: rdf.clownface({ dataset: await rdf.dataset().import(rdf.fromFile(`${dir}/manifest.ttl`)) }),
+      version: require(`${dir}/package.json`).version,
+    }
+  }
```

Comment:
Ah, might also use name from package.json itself
```suggestion
  if (hasManifest(dir)) {
    const { name, version } = require(`${dir}/package.json`)
    yield {
      name,
      manifest: rdf.clownface({ dataset: await rdf.dataset().import(rdf.fromFile(`${dir}/manifest.ttl`)) }),
      version,
    }
  }
```